test/images/agnhost/webhook/configmap.go

```diff
@@ -97,8 +97,10 @@ func mutateConfigmaps(ar v1.AdmissionReview) *v1.AdmissionResponse {
 		reviewResponse.Patch = []byte(configMapPatch2)
 	}
 
-	pt := v1.PatchTypeJSONPatch
-	reviewResponse.PatchType = &pt
+	if len(reviewResponse.Patch) != 0 {
+		pt := v1.PatchTypeJSONPatch
```

Comment:
What's the point of creating/binding `pt`? Will

```
reviewResponse.PatchType = &v1.PatchTypeJSONPatch
```
 not work?

Reply:
That's a const and we can't take address of a const.

Reply:
ack